rules/macos/process_creation/proc_creation_macos_change_file_hide_attr.yml

```diff
@@ -0,0 +1,28 @@
+title: Hidden Files Via chflags - MacOS
+id: 3b2c1059-ae5f-40b6-b5d4-6106d3ac20fe
+status: experimental
+description: Detects the use of chflags utility to hide files on MacOS
+references:
+    - https://www.sentinelone.com/labs/apt32-multi-stage-macos-trojan-innovates-on-crimeware-scripting-technique/
+    - https://attack.mitre.org/techniques/T1564/001/#:~:text=chflags
+author: Omar Khaled (@beacon_exe)
+date: 2024/08/05
+tags:
+    - attack.defense_evasion
+    - attack.t1218
+    - attack.t1564.004
+    - attack.t1552.001
+    - attack.t1105
+logsource:
+    product: macos
+    category: process_creation
+detection:
+    selection:
+        Image|endswith: /chflags
+        CommandLine|contains:
+            - 'hidden '
+            - 'nohidden '
```

Comment:
Why add `nohidden` as it's the opposite of the rule title ?

Reply:
Thanks for pointing that out @frack113 ! I added the `nohidden` flag as the certainty of using it after hiding the file is medium, but I agree it's the opposite of what the rule is about. I'll change it.